packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -73,17 +73,22 @@ def dcos_adminrouter(b, opts):
 
 @check_root
 def dcos_signal(b, opts):
-    b.cluster_id('/var/lib/dcos/cluster-id')
+    b.cluster_id()
 
 
 @check_root
 def dcos_telegraf_master(b, opts):
-    b.cluster_id('/var/lib/dcos/cluster-id')
+    b.cluster_id()
 
 
 @check_root
 def dcos_telegraf_agent(b, opts):
-    b.cluster_id('/var/lib/dcos/cluster-id', readonly=True)
+    b.cluster_id(readonly=True)
+
+
+@check_root
+def dcos_net(b, opts):
+    b.cluster_id()
```

Comment:
Is this going to be executed same way on master and agent nodes? If yes, do we need to split bootstrap into a 2 separate code paths? See `dcos_telegraf_master` and `dcos_telegraf_agent` above which does `b.cluster_id(readonly=True)` on agent.

Reply:
Yeah, either `master` or `agent` can generate the ID and rest of the nodes would then use it

Reply:
I'd split bootstrap function to `dcos_net_master` and `dcos_net_agent` and use the `b.cluster_id(readonly=True)` for agent bootstrap.

Reply:
Done